src/app/home.component.html

```diff
@@ -31,7 +31,7 @@ <h1>Select a scenario</h1>
       <div *ngIf="!loadedScenarios && !loadedCourses">Loading scenarios...</div>
     </div>
 
-    <ng-container *ngIf="scenarios.length > 0">
+    <ng-container *ngIf="scenarios && scenarios.length > 0">
```

Comment:
This should be obsolete if we land #130 first, right?

Reply:
Yes, you're right! Changes undone in favor of #130!